coreJava/src/main/java/com/brandon/corejava/data/ForzenDaoJava.java

```diff
@@ -0,0 +1,20 @@
+package com.brandon.corejava.data;
+
+import androidx.room.Dao;
+import androidx.room.Insert;
+import androidx.room.Query;
+
+import java.util.List;
+
+@Dao
+public interface ForzenDaoJava {
+
+    @Insert
+    void insert(ForzenEntityJava forzenEntityJava);
+
+    @Query("SELECT * FROM login_table")
```

Comment:
We use constants for table names and table fields

Reply:
Using a Constant for this now.